src/declarativewebutils.cpp

```diff
@@ -415,6 +416,20 @@ void DeclarativeWebUtils::handleObserve(const QString message, const QVariant da
     }
 }
 
+void DeclarativeWebUtils::updateTileSize()
```

Comment:
Actually I think that something like ```setRenderingPreferences``` would be better.

Then this method could set also:
```
    // Don't force 16bit color depth
    mozContext->setPref(QString("gfx.qt.rgb16.force"), QVariant(false));
    // Use external Qt window for rendering content
    mozContext->setPref(QString("gfx.compositor.external-window"), QVariant(true));
    mozContext->setPref(QString("gfx.compositor.clear-context"), QVariant(false));

    // Progressive painting
    mozContext->setPref(QString("layers.progressive-paint"), QVariant(true));
    mozContext->setPref(QString("layers.low-precision-buffer"), QVariant(true));
```